tests/loginInvalidUser.test.js

```diff
@@ -0,0 +1,20 @@
+const { test, expect } = require('@playwright/test');
+const { LoginPage } = require('../pages/loginPage.js');
+import { generateRandomString } from '../utils/random.js';
+
+const randomUser = 'invalidUser' + generateRandomString(4);
+const randomPassword = generateRandomString(10);
+
+test.describe('Redmine login', () => {
+    test('should not login with random user and password', async ({ page }) => {
+        const loginPage = new LoginPage(page);
+        await loginPage.goto('https://www.redmine.org/login');
```

Comment:
@karenn32 This action must be in the 'beforeEach' hook (or 'beforeAll' in case, if the test spec contains only 1 test case.).